README.md

```diff
@@ -10,6 +9,73 @@ Please refer to our [readthedocs documentation](https://deeppavlov-agent.readthe
 - Jenkins Tests Status on dev branch: [![Build Status](http://lnsigo.mipt.ru:8080/buildStatus/icon?job=assistant%2Fdev)](http://lnsigo.mipt.ru:8080/job/dp-multibranch/job/dev/)
 
 
+# Distributions
+
+We've already included two distributions: Deepy and a full-sized Dream chatbot.
+
+
+# Quick Start
+
+### Deploying via proxy
+The easiest way to try out Dream is to deploy it via proxy.
+This way all the requests will be redirected to DeepPavlov API, so you don't have to use any local resources.
+
+1. Clone the repo
+
+    ```git clone https://github.com/deepmipt/dream```
+
+2. Run
+
+    ```docker-compose -f docker-compose.yml -f dev.yml -f proxy.yml up --build```
+    
+    `dev.yml` includes volume bindings for easier debugging;
+
+    `proxy.yml` is a list of proxied containers. See [proxy usage](#proxy-usage).
+
+
+### Deploying components locally
+
+Any components which are not included in `proxy.yml` will be deployed locally.
+
+
+# Proxy usage
```

Comment:
not clear what is the difference between this section and the previous one. Maybe you need to clarify that you can exclude the debugged component from proxy, and then run